public/js/.eslintrc.json

```diff
@@ -0,0 +1,57 @@
+{
+  "root": true,
```

Comment:
We already have an eslint outside of public. Is this something that comes from AdminLte?